llama_stack/cli/model/list.py

```diff
@@ -32,6 +32,13 @@ def _add_arguments(self):
             action="store_true",
             help="Show all models (not just defaults)",
         )
+        self.parser.add_argument(
+            "-s",
+            "--search",
+            type=str,
+            required=False,
+            help="Fuzzy search the input string based on model descriptor(ID)",
```

Comment:
this is not quite "fuzzy" search, it is only "substring" search right? fuzzy search means that "llm3-8-ins" matches "llama3.1-8b-instruct", etc. can you reword?

Reply:
@ashwinb updated